broadinstitute/pearl
# AR-243 - removing duplicative titles

As part of the OurHealth survey testing feedback from yesterday, this removes the duplicative titles from the cardio history survey and lifestyle survey descriptions. The Survey title is already rendered above the survey, so we don't need to have another title.  Also, for the cario survey "Heart Health Survey" is probably better than "Cardiometabolic..." as a genreal title 

TO TEST:
1. repopulate ourhealth
2. sign into participant UI as consetend@test.com.   load the cardio and lifestyle surveys.  Confirm duplicate titles don't appear
